func/submission-created.js

```diff
@@ -47,7 +47,7 @@ export async function handler(event, context) {
                 fields: [
                     {
                         name: "Submitter",
-                        value: `<@${userInfo.id}> (${userInfo.username}#${userInfo.discriminator})`
+                        value: `<@${userInfo.id}> (${userInfo.username})`
```

Comment:
Could you add a pomelo check here?

Reply:
@sylveon okay, resolved in recent commits